data/data-pipeline/data_pipeline/etl/sources/epa_rsei_aggregate/etl.py

```diff
@@ -0,0 +1,268 @@
+                self.df[field_names.EPA_RSEI_NCSCORE_OUTPUT_FIELD].values, i
+            )
+            for i in index
+        ]
+
+        # The rationale for creating these is located here https://www.epa.gov/rsei/understanding-rsei-results#what
+        # Section: "What does a high RSEI Score mean?"
+
+        # The cut function creates equispaced bins but frequency of samples is unequal in each bin
+        # It is certainly possible to apply qcut. qcut is a quantile based function to create bins
+        # Quantile is to divide the data into equal number of subgroups or probability distributions of
+        # equal probability into continuous interval. I opted for cut as it offers greater flexibility in
+        # creating percentile bins.
+        #
+        # The result produces a cateogrical variable that affords easier stratification
+        # when performing more advanced statistical analyses of each component of the distribution. This is pertienent
+        # if we seek to combine this data with population characteristics and demographics - for example, ACS and CDC SVI Index -
+        # in order to chracterize each section of the "higher scoring" tracts for cancer-related and non-cancer related toxicity
+        self.df[field_names.EPA_RSEI_SCORE_PERCENTILE_LABEL_FIELD] = pd.cut(
+            self.df[field_names.EPA_RSEI_SCORE_OUTPUT_FIELD],
+            right=True,  # include right-most interval
+            bins=perc_func_overall_risk_score,
+            labels=labels_for_cut_function,
+            ordered=True,
+        )
+
+        self.df[field_names.EPA_RSEI_CSCORE_PERCENTILE_LABEL_FIELD] = pd.cut(
+            self.df[field_names.EPA_RSEI_CSCORE_OUTPUT_FIELD],
+            right=True,
+            bins=perc_func_cancer_score,
+            labels=labels_for_cut_function,
+            ordered=True,
+        )
+
+        self.df[field_names.EPA_RSEI_NCSCORE_PERCENTILE_LABEL_FIELD] = pd.cut(
+            self.df[field_names.EPA_RSEI_NCSCORE_OUTPUT_FIELD],
+            right=True,
+            bins=perc_func_non_cancer_score,
+            labels=labels_for_cut_function,
+            ordered=True,
+        )
+
+        # Produce percentile rank for overall risk score
+        # This was created for the sole purpose to be used in the current
+        # iteration of Score L
+        self.df[field_names.EPA_RSEI_SCORE_PERCENTILE_RANK_FIELD] = self.df[
+            field_names.EPA_RSEI_SCORE_OUTPUT_FIELD
+        ].rank(
+            ascending=True,
+            pct=True,
+        )
+
+        # This threshold was arbitrarily chosen.
+        # It would make sense to enrich this with facilities, industries, or chemical
+        # that would enable some additional form of sub-stratification when examining
+        # different percentile ranges that are derived above.
+        self.df[field_names.EPA_RSEI_SCORE_THRESHOLD_FIELD] = (
+            self.df[field_names.EPA_RSEI_SCORE_PERCENTILE_RANK_FIELD] > 0.75
```

Comment:
Want to make .75 an attribute on the class (ie., in init define `self.EPA_RSEI_SCORE_THRESHOLD_CUTOFF = .75`) to make it easier to find?